WooCommerce/Classes/ViewRelated/Dashboard/Stats v3/DashboardStatsV3ViewController.swift

```diff
@@ -0,0 +1,276 @@
+        guard let viewController = UIStoryboard.dashboard.instantiateViewController(ofClass: StoreStatsViewController.self) else {
+            fatalError()
+        }
+        return viewController
+    }()
+
+    private var newOrdersViewController: NewOrdersViewController = {
+        guard let viewController = UIStoryboard.dashboard.instantiateViewController(ofClass: NewOrdersViewController.self) else {
+            fatalError()
+        }
+        return viewController
+    }()
+
+    private var topPerformersViewController: TopPerformersViewController = {
+        guard let viewController = UIStoryboard.dashboard.instantiateViewController(ofClass: TopPerformersViewController.self) else {
+            fatalError()
+        }
+        return viewController
+    }()
+
+    // MARK: overrides
+    //
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        scrollView.refreshControl = refreshControl
+        newOrdersContainerView.isHidden = true // Hide the new orders vc by default
+
+        newOrdersViewController.delegate = self
+
+        stackView.axis = .vertical
+
+        configureContainerViews()
+        configureChildViewControllerContainerViews()
+    }
+
+    override func preferredContentSizeDidChange(forChildContentContainer container: UIContentContainer) {
+        super.preferredContentSizeDidChange(forChildContentContainer: container)
+        if let containerVC = container as? NewOrdersViewController {
+            newOrdersHeightConstraint?.constant = containerVC.preferredContentSize.height
+        }
+    }
+}
+
+extension DashboardStatsV3ViewController: DashboardUI {
+    func defaultAccountDidUpdate() {
+        storeStatsViewController.clearAllFields()
+        applyHideAnimation(for: newOrdersContainerView)
+    }
+
+    func reloadData(completion: @escaping () -> Void) {
+        let group = DispatchGroup()
+
+        var reloadError: Error? = nil
+
+        group.enter()
+        storeStatsViewController.syncAllStats() { error in
+            if let error = error {
+                reloadError = error
+            }
+            group.leave()
+        }
+
+        group.enter()
+        newOrdersViewController.syncNewOrders() { error in
+            if let error = error {
+                reloadError = error
+            }
+            group.leave()
+        }
+
+        group.enter()
+        topPerformersViewController.syncTopPerformers() { error in
+            if let error = error {
+                reloadError = error
+            }
+            group.leave()
+        }
+
+        group.notify(queue: .main) { [weak self] in
+            completion()
+            self?.refreshControl.endRefreshing()
+            if let error = reloadError {
+                DDLogError("⛔️ Error loading dashboard: \(error)")
+                self?.handleSyncError(error: error)
+            } else {
+                self?.updateSiteVisitStatsVisibility(shouldShowSiteVisitStats: true)
```

Comment:
I know this part wasn't yours. What do you think of renaming this method to `showSiteVisitors(true)`? It's pretty clear from the view controller's context that this is a stat.